extensions/registration-service/src/main/java/org/eclipse/dataspaceconnector/registration/api/RegistrationService.java

```diff
@@ -90,6 +93,7 @@ public void addParticipant(String did) {
         var participant = Participant.Builder.newInstance()
                 .did(did)
                 .status(ONBOARDING_INITIATED)
+                .traceContext(telemetry.getCurrentTraceContext())
```

Comment:
Here, shouldn’t there be UT coverage that we are saving the context etc

Reply:
I see, I updated a test in RegistrationServiceTest to include the traceContext